src/screens/discover/components/DiscoverSearch.js

```diff
@@ -28,7 +28,7 @@ import {
   useSwapCurrencyList,
 } from '@/hooks';
 import { useNavigation } from '@/navigation';
-import Routes from '@/navigation/routesNames';
+import Routes from '@rainbow-me/routes';
```

Comment:
Nit: this was correct before, should be `@/navigation/routesNames`

Reply:
Will fix that back, probably pressed suggestion from my IDE or something like that